assets/css/app.css

```diff
@@ -0,0 +1,68 @@
+body {
+  font-family: 'Poiret One', cursive;
+  font-size 1.6em
+}
+
+.form__button--submit {
+  width: 15%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 0.5em;
+}
+
+.filters__button {
+  text-align: center
+}
+
+.filters__button--active   {
+  width: 15%;
+}
+
+.filters__button--all {
+  width: 15%;
+}
+
+.filters__button--complete {
+  width: 15%;
+}
```

Comment:
BEM should mean that anything with a `filters__button--complete` class **also** has a `filters__button` class. That means that any common styles (like the repeated `width` here) can be DRYed up to the Element class, rather than repeated throughout Modifier classes.